ember-intl/addon/services/intl.d.ts

```diff
@@ -1,6 +1,6 @@
-import type { EmberRunTimer } from '@ember/runloop/types';
+import type { Timer as EmberRunTimer } from '@ember/runloop';
 import Service from '@ember/service';
-import type { SafeString } from '@ember/template/-private/handlebars';
+import type { SafeString } from '@ember/template';
```

Comment:
Something that may help solve the type problem more easily: I want to remove `SafeString` as a possible return type. That is, we assume that all helpers from `ember-intl` return `string`.

The error from `Timer as EmberRunTimer` might go away, if I can change `addon/services/intl.js` to `addon/services/intl.ts` (i.e. derive the types instead of manually defining a declaration file).

Reply:
I haven't forgotten about this, just haven't had motivation to work on `ember-intl` since the year. I will get to it some day. 🙂